motoko/classes/Makefile

```diff
@@ -0,0 +1,29 @@
+.PHONY: all
+all: build
+
+.PHONY: build
+.SILENT: build
+build:
+	dfx canister create --all
+	dfx build
+
+.PHONY: install
+.SILENT: install
+install: build
+	dfx canister install --all
+
+.PHONY: upgrade
+.SILENT: upgrade
+upgrade: build
+	dfx canister install --all --mode=upgrade
+
+.PHONY: test
+.SILENT: test
+test: install
+	dfx canister call Test run '()' \
+		| grep '()' && echo 'PASS'
+
+.PHONY: clean
+.SILENT: clean
+clean:
+	rm -fr .dfx
```

Comment:
I usually write `@ rm ...` and then the `.SILENT` is  not needed. For one-liners.

Reply:
thanks - I'm just cargo-cult coding here so will keep to same style.